src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+    25%         2.490000
+    50%         3.990000
+    75%         7.490000
+    max        59.990000
+    Name: price, dtype: float64
+
+
+
+    
+![png](EDA_1_files/EDA_1_11_1.png)
+    
+
+
+    
+    Frecuencia de categorías de productos:
+    product_type
+    cleaning-products                  160
+    tins-packaged-foods                125
+    snacks-confectionery               122
+    cooking-ingredients                 73
+    pasta-rice-noodles                  66
+    baby-toddler-food                   62
+    condiments-dressings                52
+    cereal                              51
+    haircare                            50
+    biscuits-crackers                   49
+    soft-drinks-mixers                  48
+    baby-kids-toiletries                45
+    skincare                            44
+    cooking-sauces                      43
+    dog-food                            42
+    dental                              42
+    baby-accessories                    41
+    superfoods-supplements              37
+    cat-food                            35
+    shaving-grooming                    29
+    nappies-nappy-pants                 29
+    dishwashing                         27
+    period-care                         26
+    home-baking                         25
+    hand-soap-sanitisers                24
+    long-life-milk-substitutes          24
+    tea                                 22
+    medicines-treatments                20
+    spreads                             19
+    spirits-liqueurs                    19
+    wine                                18
+    toilet-roll-kitchen-roll-tissue     18
+    bath-shower-gel                     18
+    washing-liquid-gel                  18
+    fabric-softener-freshener           17
+    baby-milk-formula                   16
+    bin-bags                            15
+    coffee                              15
+    food-bags-cling-film-foil           14
+    deodorant                           13
+    beer                                11
+    low-no-alcohol                       9
+    maternity                            9
+    suncare                              9
+    washing-powder                       8
+    sexual-health                        8
+    washing-capsules                     8
+    delicates-stain-remover              8
+    pet-care                             7
+    adult-incontinence                   7
+    mixed-bundles                        6
+    medicine-treatments                  5
+    drying-ironing                       5
+    premixed-cocktails                   5
+    cider                                4
+                                         4
+    household-sundries                   3
+    other-hot-drinks                     2
+    water-softener                       2
+    Name: count, dtype: int64
+    
+    Frecuencia de etiquetas de productos:
+    Series([], Name: count, dtype: int64)
```

Comment:
What is the conclusion? An EDA is not simply plots and tables, you should be making "a story": "We want to check X, Y and we hypothesis H and I --> we plot / explore this --> our conclusion is C"